src/modules/utils/globals.js

```diff
@@ -5,9 +5,16 @@ export const get = ( key, defaultValue ) => window[ key ] || defaultValue;
 export const google = () => get( 'google' );
 export const wpApi = wp.api;
 export const wpApiRequest = wp.apiRequest;
+export const wpComponents = wp.components;
 export const wpData = wp.data;
+export const wpEditor = wp.blockEditor || wp.editor;
 export const wpHooks = wp.hooks;
 
+export const wpDataSelectCoreEditor = () => wpData.select( 'core/block-editor' ) ||
+	wpData.select( 'core/editor' );
+
+export const IconButton = wpComponents.Button || wpComponents.IconButton;
```

Comment:
I think this is OK for now, but I wonder if this belongs in another place other than here